packages/graphql-mocks/src/resolver-map/layer.ts

```diff
@@ -37,7 +38,11 @@ export function layer(partials: ResolverMap[], options?: LayerOptions): Resolver
       let resolver = getResolver(layeredResolverMap, reference);
 
       if (resolver && options?.wrappers?.length) {
-        resolver = await applyWrappers(resolver, options.wrappers, {
+        // if there is at least one wrapper, the embedPackOptionsWrapper should also
+        // be included in order to better manage the context and ensure that the `pack`
+        // property is included
+        const wrappers = [embedPackOptionsWrapper, ...(options?.wrappers ?? [])];
```

Comment:
Set embedPackOptionsWrapper before any other wrappers so that it can ensure pack is included on the context in case wrappers after it drop it somehow.